salt/modules/beacons.py

```diff
@@ -251,3 +251,93 @@ def disable(**kwargs):
             # Effectively a no-op, since we can't really return without an event system
             ret['comment'] = 'Event module not available. Beacons enable job failed.'
     return ret
+
+
+def enable_beacon(name, **kwargs):
+    '''
```

Comment:
I can.  They were all added new for Beryllium though and there is a verionadded at the top of the module, should I add the versionadded for all of them?